torchvision/models/detection/transform.py

```diff
@@ -56,6 +62,17 @@ def normalize(self, image):
         return (image - mean[:, None, None]) / std[:, None, None]
 
     def resize(self, image, target):
+        if self.is_xla:
+            # Resize to a fixed size on TPU to avoid dynamicity.
+            new_h, new_w = 600, 800
```

Comment:
should these be command line arguments, or the first size the class sees, instead of fixed values?

Reply:
Think making those configurable is definitely a better idea yeah, although might have to be done in another PR since I'm not sure I'll be able to have time to do this